app/views/manifestation/_involved_authorities.html.haml

```diff
@@ -0,0 +1,17 @@
+%h3= t(:associated_persons)
```

Comment:
I've created this partial to replaces old shared/associated_people partial to reduce code duplication